addons/website_event/views/event_type_views.xml

```diff
@@ -6,31 +6,20 @@
         <field name="model">event.type</field>
         <field name="inherit_id" ref="event.view_event_type_form"/>
         <field name="arch" type="xml">
-            <div name="event_type_visibility_seats" position="after">
-                <div class="col-12 col-lg-6 o_setting_box" name="event_type_visibility_website">
-                    <div class="o_setting_left_pane">
+            <xpath expr="//div[hasclass('oe_title')]" position="after">
+                    <span name="website_menu">
+                        <label for="website_menu" string="Website Submenu"/>
                         <field name="website_menu"/>
-                    </div>
-                    <div class="o_setting_right_pane">
-                        <label for="website_menu"/>
-                        <div class="row" name="website_menu">
-                            <div class="col-lg-8 mt16 text-muted">
-                                Check this option to have menus for your event on the
-                                website: registrations, schedule, map, ...
-                            </div>
-                        </div>
-                        <div class="row mt16" name="menu_register_cta"
-                            groups="base.group_no_one">
-                            <label class="col" for="menu_register_cta"/> <field name="menu_register_cta"/>
-                        </div>
-                        <div class="row mt16" name="community_menu"
-                            id="community-menu"
-                            attrs="{'invisible': [('website_menu', 'in', (True, False))]}">
-                            <label class="col" for="community_menu"/> <field name="community_menu"/>
-                        </div>
-                    </div>
-                </div>
-            </div>
+                    </span>
+                    <span id="community-menu" attrs = "{'invisible': [('website_menu', '=', False)]}">
```

Comment:
attrs has changed. Is it normal ? Noting that you hide field in bridge modules but the complete span is already invisible  if ('website_menu', '=', False).

Reply:
After taking a deeper look at this, it appears that the community_menu field in website_event module is just a placeholder that other sub modules could use (website_event_meet, website_event_track_quiz), so we can leave this unchanged and keep `{'invisible': [('website_menu', 'in', (True, False))]}`